javascriptv3/scripts/test.sh

```diff
@@ -1,27 +1,12 @@
 #!/bin/bash
 
-# Set valid arguments.
-valid_args=("@unit" "@integration")
-
-# Iterate over all arguments.
-for arg in "$@"; do
-  # Check if the current argument is a valid argument.
-  if [[ " ${valid_args[@]} " =~ " ${arg} " ]]; then
-    # If the argument is "@unit", run `lerna run test`.
-    if [ "$arg" == "@unit" ]; then
-      lerna run test
-    # If the argument is "@integration", run `lerna run integration-test`.
-    elif [ "$arg" == "@integration" ]; then
-      lerna run integration-test
-    fi
-  else
-    # If the argument is invalid, print an error message.
-    echo "Invalid argument: $arg"
-  fi
-done
-
-# If no valid arguments are provided, run both `lerna run test` and `lerna run integration-test`.
-if [ "$#" -eq 0 ]; then
+if [ "$TEST_SCOPE" == "unit" ]; then
   lerna run test
+elif [ "$TEST_SCOPE" == "integration" ]; then
   lerna run integration-test
```

Comment:
Is `lerna run integration-test` your existing command for just running integration tests?